README.md

```diff
@@ -32,7 +32,7 @@ To help with testing app updates before they're released, you can:
 
 ### Contribute Code 
 
-1. Look in [GitHub issues](https://github.com/mattermost/mattermost-server/issues?q=label%3A"React+Native") for issues marked as [Help Wanted]
+1. Look in [GitHub issues](https://github.com/mattermost/mattermost-server/issues?utf8=%E2%9C%93&q=label%3A%22Help+Wanted%22+label%3A%22Tech%2FReact+Native%22+is%3Aopen+is%3Aissue) for issues marked as [Help Wanted]
```

Comment:
Would you please use `https://mattermost.com/pl/help-wanted-mattermost-mobile` instead? It's a permalink that redirects to the appropriate GH URL.

Reply:
Yea sure, updated as suggest